[2024-03-25T19:12:02.783+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_get_up_gcs.upload_to_gcs manual__2024-03-25T19:11:50.247729+00:00 [queued]>
[2024-03-25T19:12:02.813+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_get_up_gcs.upload_to_gcs manual__2024-03-25T19:11:50.247729+00:00 [queued]>
[2024-03-25T19:12:02.815+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-25T19:12:02.860+0000] {taskinstance.py:2217} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_to_gcs> on 2024-03-25 19:11:50.247729+00:00
[2024-03-25T19:12:02.873+0000] {standard_task_runner.py:60} INFO - Started process 4562 to run task
[2024-03-25T19:12:02.884+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_get_up_gcs', 'upload_to_gcs', 'manual__2024-03-25T19:11:50.247729+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/test_get_up_gcs.py', '--cfg-path', '/tmp/tmph17r5kd8']
[2024-03-25T19:12:02.890+0000] {standard_task_runner.py:88} INFO - Job 47: Subtask upload_to_gcs
[2024-03-25T19:12:03.018+0000] {task_command.py:423} INFO - Running <TaskInstance: test_get_up_gcs.upload_to_gcs manual__2024-03-25T19:11:50.247729+00:00 [running]> on host c5a6ac367c49
[2024-03-25T19:12:03.266+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_get_up_gcs' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2024-03-25T19:11:50.247729+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-25T19:11:50.247729+00:00'
[2024-03-25T19:12:03.291+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-03-25T19:12:03.303+0000] {base.py:83} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-03-25T19:12:04.374+0000] {gcs.py:557} INFO - File /opt/***/dags/data_get.json uploaded to data/data_get.json in ds525-capstone-test-49 bucket
[2024-03-25T19:12:04.400+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=test_get_up_gcs, task_id=upload_to_gcs, execution_date=20240325T191150, start_date=20240325T191202, end_date=20240325T191204
[2024-03-25T19:12:04.461+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-25T19:12:04.513+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
